public/src/app/file-upload/file-upload.component.ts

```diff
@@ -1,9 +1,12 @@
 //
 // inspired by http://www.advancesharp.com/blog/1218/angular-4-upload-files-with-data-and-web-api-by-drag-drop
 //
-import { Component, Input, Output, EventEmitter, ViewChild, ElementRef, HostListener } from '@angular/core';
```

Comment:
We most likely don't use a file upload component in the public front-end, so I'm assuming this can be removed.

Reply:
Nice spot on this, I had doubt with this one before (but admin uses it), I will check again if it really is not being used will remove this one.